### ISSUE TITLE
cmd.ctrld_exits default wrong in docs?

### ISSUE THREAD

**Issue by elieux** [chrisant996/clink#238]
Recently after upgrading (from an earlier v1.3.x I believe), Ctrl+D stopped working for me. If I add `cmd.ctrld_exits = true` to my setting file, it starts working again. According to the [docs](https://chrisant996.github.io/clink/clink.html#configuring-clink), it should still be on by default though. Am I reading it wrong?

**Comment by chrisant996** [comment]
You installed with the "Enable enhanced default settings".  Certain settings have two possible defaults.

I'll see if there's a way to clarify that without getting overly complicated/verbose.

**Comment by chrisant996** [comment]
Also, I am planning to rename the default_settings and default_inputrc files in the .zip file so that they will only take effect if manually renamed.  (Those are the files that correspond to the "Enable enhanced default settings" checkbox in the setup exe installer.)

**Comment by elieux** [comment]
Ah, that explains why they took effect even though I don't use the installer. I read about the option in the release notes but didn't think it would apply. I'll be looking forward to the change.

**Comment by elieux** [comment]
As for the docs, it kinda makes sense now. My first idea would be to just add a mark (e.g. an asterisk) to the settings that are changed by "Enable enhanced default settings" and add a note below the table.

**Comment by wswind** [comment]
this is where cause this problem,
```
#in the file <install-dir>\default_settings 
cmd.ctrld_exits                   = False # change this to True would fix my problem
```
I think this config should not be False in the config file released. People who upgrade this version would be confused about why ctrl +d is suddenly not working

**Comment by chrisant996** [comment]
Docs are updated as well. Commit 0db368bb5990eddc65ada19f6d63790d186dba9b.

**Comment by koppor** [comment]
Thank you for updating. Would it be possible to have `cmd.ctrld_exits` set to `True` in `default_settings`? If not, I would like to learn about the decision driver to change the default behavior.

Background: I use the chocolatey package `clink-maintained` and after one auto update, clink looked more nice, but <kbd>Ctrl</kbd>+<kbd>D</kbd> and <kbd>Ctrl</kbd>+<kbd>A</kbd> stopped working as expected. Thanks to the linked diff, I learned that I need to put `clink.default_bindings = bash` into `default_settings`. - For the googlers, please read on at https://chrisant996.github.io/clink/clink.html#configuring-clink.

**Comment by chrisant996** [comment]
> Thank you for updating. Would it be possible to have `cmd.ctrld_exits` set to `True` in `default_settings`? If not, I would like to learn about the decision driver to change the default behavior.

No, that would ruin the whole point of the "enhanced defaults".  The purpose of the "enhanced defaults" is to make Clink behave like CMD/Windows, instead of like bash/Linux.  When you press Ctrl+D in CMD on Windows, it does not terminate the process.  So, no, from my perspective setting `cmd.ctrld_exits` to `true` in `default_settings` would ruin the whole point.

The issue where "enhanced defaults" were accidentally enabled when installing from .zip file has already been fixed by f5bb747cf6e73c7d6930c3143c55e5aea04ccc2d.  What you are encountering is a leftover effect:  The clink-maintained installer unzips the .zip file, right?  It probably doesn't have a way to delete files.  You probably have _both_ `default_settings` and `_default_settings`.  I can't do anything about that at this point; what's done is done.  I have already made it so the problem won't happen again in the future.

So, what is the motivation for the request?

1. Do you intentionally want to use most of the "enhanced defaults", but you dislike that one setting?
2. Do you not really want to use "enhanced defaults", and you noticed that one of the things affected by "enhanced defaults" is the Ctrl+D behavior?

If (1), then the best thing is to run `clink set cmd.ctrld_exits true`.

If (2), then the best thing is to delete `default_settings` and `default_inputrc`.
